### DISCUSSION TITLE
Supabase UI - deleting rows?

### DISCUSSION THREAD

**Discussion by march08** [supabase/supabase#1777] [Questions] [answered]
![Screen Shot 2021-05-31 at 13 42 02](https://user-images.githubusercontent.com/15802823/120188169-1443f880-c216-11eb-92ad-0f4f16433deb.png)
I am not sure if I am blind and I just can't find the button but how do I delete a row?
The keyboard shortcuts also doesn't seem to work. Thanks

**Comment by inian** [comment, marked as answer]
Hi @march08, this is a high priority bug we are fixing now! The option can be accessed by clicking on the dropdown here. 
![image](https://user-images.githubusercontent.com/2155545/120189252-aca7a080-c249-11eb-9104-61f3d460794a.png)

**Reply by march08** [reply]
Ah! Ok totally missed that arrow. Thanks!
